specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdnwebapplicationfirewall.json

```diff
@@ -599,6 +599,7 @@
           "type": "string",
           "enum": [
             "RemoteAddr",
+            "SocketAddr",
             "Country",
```

Comment:
Same as #PR 8573, Should this be removed too?

Reply:
Yes. "Country" had to be removed. Removed it now.